src/components/Explore/TracesByService/MiniREDPanel.tsx

```diff
@@ -113,13 +113,13 @@ export class MiniREDPanel extends SceneObjectBase<MiniREDPanelState> {
 
   private getRateOrErrorPanel(metric: MetricFunction) {
     const panel = barsPanelConfig().setHoverHeader(true).setDisplayMode('transparent');
-    if (metric === 'errors') {
-      panel.setTitle('Errors rate').setCustomFieldConfig('axisLabel', 'Errors').setColor({
+    if (metric === 'rate') {
+      panel.setCustomFieldConfig('axisLabel', 'span/s');
```

Comment:
Is it always `span/s` or can be `span/min`, `span/h`, etc. depending on final resolution?

Reply:
Hmm good question. I'm not actually [sure](https://github.com/grafana/traces-drilldown/blob/main/src/components/Explore/queries/generateMetricsQuery.ts#L54). @adrapereira do you have any insight here?